src/runtime-rs/crates/resource/src/rootfs/mod.rs

```diff
@@ -58,10 +58,13 @@ impl RootFsResource {
                 // Safe as single_layer_rootfs must have one layer
                 let layer = &mounts_vec[0];
 
-                let rootfs = if let Some(_dev_id) = get_block_device(&layer.source) {
+                // not support at this stage
+                /*let rootfs = if let Some(_dev_id) = get_block_device(&layer.source) {
```

Comment:
Commented out code.

Reply:
Thanks for your review, delete this code.